accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/installhook/impl/AcToolInstallHookServiceImpl.java

```diff
@@ -27,17 +29,24 @@
 @Component
 public class AcToolInstallHookServiceImpl implements AcToolInstallHookService {
 
+    public static final String ACL_HOOK_PATHS="actool.hook.config.paths";
     @Reference(policyOption = ReferencePolicyOption.GREEDY)
     private AcInstallationServiceInternal acInstallationService;
 
     @Reference(policyOption = ReferencePolicyOption.GREEDY)
     private ConfigFilesRetriever configFilesRetriever;
 
     @Override
-    public PersistableInstallationLogger installYamlFilesFromPackage(Archive archive, Session session, ProgressTrackerListener progressTrackerListener)
+    public PersistableInstallationLogger installYamlFilesFromPackage(VaultPackage  vaultPackage, Session session, ProgressTrackerListener progressTrackerListener)
             throws Exception {
+        Archive archive=vaultPackage.getArchive();
```

Comment:
nitpick: space before and after `=`. Please check your formatter. This is wrong in all new lines.

Reply:
thanks! I used a new workspace and forgot to configure it.